notebooks/en/fine_tuning_llm_for_generate_persian_product_catalogs_in_json_format.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>Maybe explain SFTTrainer just in one sentence, people might not know and some params like packing or formatting function that we normally don't have in Trainer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>